web/templates/user_custom_profile_fields.hbs

```diff
@@ -4,7 +4,12 @@
             <div class="name">{{this.name}}</div>
         {{/unless}}
         {{#if this.is_link}}
-            <a tabindex="0" href="{{this.value}}" target="_blank" rel="noopener noreferrer" class="value custom_profile_fields_link {{#if ../for_user_info_popover}}tippy-zulip-tooltip{{/if}}" data-tippy-content="{{this.name}}">{{this.value}}</a>
+            <div class="custom_user_URL_field">
+                <a tabindex="0" href="{{this.value}}" target="_blank" rel="noopener noreferrer" id="custom_profile_fields_long_url" class="value custom_profile_fields_link {{#if ../for_user_info_popover}}tippy-zulip-tooltip{{/if}}">{{this.value}}</a>
+                <button type="submit" id="copy_custom_field_url" class="tippy-zulip-tooltip" data-tippy-content="{{t 'Copy URL' }}">
+                    <i class="fa fa-clipboard copy-blue"></i>
```

Comment:
I think we should be using `{{> copy_to_clipboard_svg }}` here instead. Also, another commit to replace it in `bot_avatar_row.hbs` would be nice so that we use the same copy icon across the app.